scss/components/_banner.scss

```diff
@@ -14,10 +14,18 @@
         border-bottom: none;
     }
 
+    &__container {
+        display: flex;
```

Comment:
Only partially supported by IE11. We probably want to change it so it works, or use the ms prefix

Reply:
I've added the prefixes in; here is how it looks at medium viewport size on IE11 with the flex properties. The partial support, from what I could see on `caniuse`, was because of bugs related to the translation of `flex: 1` as well as `flex-direction: column` when a `min-height` is specified. I think the banner looks fine because we're not using those properties here. Happy to change to something non-flex if you think it best, though

![image](https://user-images.githubusercontent.com/23668262/74038902-d5eb5000-49b8-11ea-8b40-5424354e8bb2.png)